data-pipeline/src/pipeline/main.py

```diff
@@ -111,8 +111,8 @@ def pre_process_ks4(run_type, year) -> pd.DataFrame:
 
 def pre_process_academy_ar(run_type, year) -> tuple[pd.DataFrame, pd.DataFrame]:
     logger.info("Processing Academy AR Data")
-    academy_ar_data = get_blob(raw_container, f"{run_type}/{year}/academy_ar.xlsx")
-    aar = prepare_aar_data(academy_ar_data)
+    academy_ar_data = get_blob(raw_container, f"{run_type}/{year}/aar.csv")
```

Comment:
Looks like this doesn't match the name of the files in `raw`, same with the `aar_cs.csv` file as well.

Reply:
Note: In the list of required files, I'll keep the names as aar and aar_cs, so we should just change the name of the file we have in blob store to resolve this.